### PR TITLE
concise problem statement

### PR DESCRIPTION
**Pull request by ivanov**
thanks to a conversation with @dcoates

### CODE REVIEW

File: doc/design.rst [hunk 1]
```diff
@@ -11,6 +11,14 @@ The IPython notebook is pleasantly interactive in showing us rendering of LaTeX,
 and output of code execution, but the web interface feels heavy and
 uncomfortable when writing anything longer than a few pages.
 
+*******************************
+What's wrong with the notebook?
+*******************************
+
+Using model–view–controller (MVC) terminology, the Jupyter notebook interface
```
Comment (matthew-brett) [inline doc/design.rst]: More detail on what the model, view and controller are in terms of the notebook?

Comment (ivanov) [inline doc/design.rst]: not sure if i explained it better, but gave it a shot.

**Comment by matthew-brett** [comment]
Yeah - that's better, thanks.

**Comment by matthew-brett** [comment]
Incidentally, the notebook can also become unwieldy as a view.

This notebook :  https://github.com/matthew-brett/perrin-academy/blob/0d29edc81b6910fbca9d3ab9dc4651941bc6daa5/fourier_no_ei.ipynb - takes 17 seconds for the math to render, serving mathjax from a local copy.

**Pull request merged**